phpBB/includes/mcp/mcp_main.php

```diff
@@ -1002,6 +1024,28 @@ function mcp_delete_topic($topic_ids, $is_soft = false, $soft_delete_reason = ''
 			$s_hidden_fields['delete_permanent'] = '1';
 		}
 
+		/**
+		* This event allows you to modify the hidden form fields when deleting topics
+		*
+		* @event core.mcp_delete_topic_modify_hidden_fields
+		* @var	string	l_confirm				The mode we are deleting in (DELETE_TOPIC(S), DELETE_TOPIC(S)_PERMANENTLY)
```

Comment:
This is actually the confirmation text language variable

Reply:
I added the DELETE_TOPIC(S) etc for usage context if looking up the event. Should I change the whole thing or just the text before it to reflect that it's the confirmation text language variable?

Reply:
Changing just the text before the possible values is fine